translations/it.js

```diff
@@ -5,28 +5,28 @@
   it = {
     topBarHeading: 'Un progetto Zooniverse',
     signUpHeading: 'Crea un profilo Zooniverse',
-    signInHeading: 'Accedi al tuo profilo Zooniverse',
+    signInHeading: 'Accedi a Zooniverse',
     signUp: 'Registrati',
     signIn: 'Accedi',
     signOut: 'Esci',
-    username: 'Username',
+    username: 'Nome utente',
     password: 'Password',
     email: 'Email',
     realName: 'Nome reale',
-    whyRealName: 'Questo verra\' utilizzato per ringraziare i collaboratori, ad esempio, in presentazioni e posters. Se non vuoi che il tuo nome venga utilizzato in pubblico, lascia questo spazio vuoto.',
+    whyRealName: 'Questo dato serve per ringraziare e citare i volontari: ad esempio, in presentazioni e poster. Se non vuoi che il tuo nome venga utilizzato in pubblico, lascia questo spazio vuoto.',
     noAccount: 'Non hai un profilo?',
-    agreeToPrivacyPolicy: 'Accetto <a href="https://www.zooniverse.org/privacy" target="_blank">le norme sulla Privacy</a>.',
-    betaPreference: 'Vorrei ricevere notifiche per testare e commentare su progetti Zooniverse non ancora pubblicati.',
-    forgotPassword: 'Hai dimenticato la tua password?',
-    badLogin: 'Username or password incorretti',
-    signInFailed: 'Errore di registrazione',
-    signInForProfile: 'Accedi al tuo profilo.',
-    footerHeading: 'Zooniverse è una raccolta di progetti web di Citizen Science basati sull\'utilizzo di volontari che aiutano i ricercatori nel gestire flussi di dati.',
+    agreeToPrivacyPolicy: 'Accetto le <a href="https://www.zooniverse.org/privacy" target="_blank">Politiche sulla privacy</a>.',
```

Comment:
@SebastianoPistore grazie per la PR. Perche' `Politiche` e' maiuscolo?

Reply:
L'ho scritto così per evidenziare il collegamento (far capire all'utente che cliccando in quel punto è possibile leggerle le condizioni), visto che nessuno lo fa mai.

Reply:
Utilizzare maiuscole per evidenZiare links non e’ una procedura web standard. HTML tags sono utilizzate per quello. Ti spiacerebbe modificare la maiuscola per favore?